notebook/static/notebook/js/outputarea.js

```diff
@@ -659,9 +659,9 @@ define([
     var append_text = function (data, md, element) {
         var type = 'text/plain';
         var toinsert = this.create_output_subarea(md, "output_text", type);
+        data = utils.fixCarriageReturn(data);
```

Comment:
Is this reordering required? I'm not objecting to it, just wondering if there was a bug caused by the previous order.